scripts/db_migrator.py

```diff
@@ -984,10 +984,19 @@ def version_3_0_5(self):
     def version_3_0_6(self):
         """
         Version 3_0_6
-        This is the latest version for 202211 branch
         """
 
         log.log_info('Handling version_3_0_6')
+        self.set_version('version_3_0_7')
```

Comment:
I think the change here should reflect the real state of version 3_0_6. In other words, the PR that added 3_0_6 to 202205 should also have updated master branch to avoid running into this issue in the first place.

IMO the migration logic added in the PR https://github.com/sonic-net/sonic-utilities/pull/2870 should be updated here.

Reply:
This is already handled as part of version 4_0_2 in master and 202305 through this PR https://github.com/sonic-net/sonic-utilities/pull/2839. Hence we may just want to add a dummy version of 3_0_6  to avoid db_migrator failure.